### PR TITLE
fixes #12783 - make community-templates source for job templates

### PR DESCRIPTION
**Pull request by stbenjam**
This changes the way we store job templates to the same way Foreman does it. https://github.com/theforeman/community-templates/pull/268 puts the templates in jobs/ directory.

At release time, the releaser will sync the templates. If you'd like to test the sync_templates script, you can put it against my fork.

### CODE REVIEW

File: scripts/sync_templates.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+REPO=$(mktemp -d)
+trap "rm -rf $REPO" EXIT
+
+git clone -q -b develop https://github.com/theforeman/community-templates $REPO/ct
+
+# Move into destination dir if run from plugin root
+[ -d app/views/templates ] && cd app/views/templates
+
+# Add underscore prefix to snippets
+[ -d $REPO/ct/jobs/snippets ] && (cd $REPO/ct/jobs/snippets && prename -f 's/^([^_])/_$1/' *)
```
Comment (adamruzicka) [inline scripts/sync_templates.sh]: Is `prename` a typo?

File: scripts/sync_templates.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (adamruzicka) [inline scripts/sync_templates.sh]: Is `bash` necessary? Couldn't it be run by plain old `sh`?

File: scripts/sync_templates.sh (hunk 1 above)
Comment (stbenjam) [inline scripts/sync_templates.sh]: No, it's a bash utility to prefix files

File: scripts/sync_templates.sh (hunk 2 above)
Comment (stbenjam) [inline scripts/sync_templates.sh]: This is just a lightly modified copy of foreman's script

Comment (stbenjam) [inline scripts/sync_templates.sh]: It works with just sh so I updated it

File: scripts/sync_templates.sh (hunk 1 above)
Comment (stbenjam) [inline scripts/sync_templates.sh]: Oh, sorry, it's some utility bundled with perl on Debian only.  I forget how I got it on my Fedora box, it wasn't packaged.  It's not a bug deal, I'll rewrite it something more portable.

Comment (adamruzicka) [inline scripts/sync_templates.sh]: Oh, I see. It seemed odd to me not having it on my box

Comment (stbenjam) [inline scripts/sync_templates.sh]: Made it just a for loop.

**Comment by stbenjam** [comment]
[test]

**Comment by stbenjam** [comment]
[test]

**Comment by ares** [comment]
bad luck :-) let me try [test]

**Comment by ares** [comment]
[test]

**Comment by stbenjam** [comment]
Hah, can we put the "merge anyway" policy back in place for transient failures? :)

**Comment by ares** [comment]
I guess we can :-) This is the last try :-)

**Comment by stbenjam** [comment]
Good to go?

**Comment by ares** [comment]
Sorry, somehow it got off my radar. Thanks @stbenjam, merging!

**Pull request merged**